modules/acs-secured-cluster-services-overview.adoc

```diff
@@ -14,5 +14,5 @@ Sensor also triggers deploy-time and runtime violations based on {product-title-
 * *Admission controller*: The Admission controller prevents users from creating workloads that violate security policies in {product-title-short}.
 * *Collector*: Collector analyzes and monitors container activity on cluster nodes.
 It collects container runtime and network activity information and sends the collected data to Sensor.
-* *Scanner*: On Kubernetes, the Secured cluster services do not include Scanner.
-However, on the {ocp}, {product-title-short} installs a lightweight Scanner version on each secured cluster to scan images in the integrated {ocp} registry.
+* *Scanner*: In Kubernetes, the secured cluster services include Scanner-slim as an optional component.
+However, on the {ocp}, {product-title-short} installs a lightweight Scanner-slim version on each secured cluster to scan images in the integrated {ocp} registry.
```

Comment:
With the changes in ACS 4.1 that allows for 'delegated scanning', the lightweight scanner in OCP can scan more than just the integrated OCP registry. 

Consider changing this message to in indicate more than just the integrate ocp registry, something along the lines of: `... scan images in the integrated {ocp} registry (and optionally other registries)`.